src/main/java/mate/academy/dto/UpdateBookRequestDto.java

```diff
@@ -1,14 +1,37 @@
 package mate.academy.dto;
 
+import jakarta.validation.constraints.DecimalMin;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Size;
 import java.math.BigDecimal;
 import lombok.Data;
+import mate.academy.validator.CoverImage;
+import mate.academy.validator.Isbn;
 
 @Data
 public class UpdateBookRequestDto {
+    @NotBlank(message = "Title cannot be blank")
+    @Size(max = 20, message = "Title length must not exceed {max} characters")
     private String title;
+
+    @NotBlank(message = "Author cannot be blank")
+    @Size(max = 30, message = "Author length must not exceed {max} characters")
     private String author;
+
+    @Isbn
     private String isbn;
+
+    @NotNull(message = "Price is mandatory")
+    @DecimalMin(value = "0.0", inclusive = false,
+            message = "Price must be greater than zero")
     private BigDecimal price;
+
+    @NotNull(message = "Description is mandatory")
```

Comment:
not null doesn't make sense - if it's optional, let it be nullable, if it's required - it should be notBlank